assets/stylesheets/main.css

```diff
@@ -0,0 +1,16 @@
+.bold-text {
+  font-weight: bold;
+}
+
+.remove-btn {
+  max-height: 40px;
+  max-width: 35px;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.table > :not(caption) > * > * {
+  border: 0;
+}
```

Comment:
- [ ] Please follow the given layouts check how the design looks like on this image
![lay](https://user-images.githubusercontent.com/44745855/133476812-64a4efe1-6b14-4986-8a94-4ba40d70ca47.png)

your page
![aw](https://user-images.githubusercontent.com/44745855/133476884-a9914c79-2c3c-4b18-93c4-47eafc897073.png)

As you can see layouts are differents please make necessary changes

Reply:
Hey, thanks for the review, can we keep the table not displaying anything if it's empty?

Reply:
Hey @leolpaz,

Yes a table will display something after adding a book just follow the layouts only